server/controllers/index.js

```diff
@@ -8,75 +8,81 @@ let currentUsers = [];
 
 /* Check if player is logged in */
 router.get('/auth', (req, res) => {
-    if (req.session.user) {
+    if (req.session.user) { /* The user session exists, authorised */
```

Comment:
could use destructuring here
`
const { user } = req.session;
`
then use the user var line 11 and 12.

Reply:
What does this do exactly? Does it match the field in the object on the right (in our case user) and assign it to our const?

Reply:
yeah exactly, its called destructuring, you can do it with an array or object
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
right, okay :) thanks